bower.json

```diff
@@ -28,6 +28,16 @@
     "/demo/"
   ],
   "dependencies": {
-    "polymer": "^1.7.0"
+    "polymer": "1 - 2"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "^1.7.0"

```

Comment:
Could be `^1.9.1`, not that it overly matters granted

Reply:
I figured I'd point it at the previous version, but `^1.9.1` is valid :+1: